### PR TITLE
Last week

### PR DESCRIPTION
**Pull request by hachi-ops**
Some rendering of the components started. Initially I am rendering everything on the dashboard, later I will be rendering on button click.

### CODE REVIEW

File: client/src/components/dashboard/NewMessage.js [hunk 1]
```diff
@@ -23,28 +23,35 @@ function NewMessage() {
...
-          onChange={(e) => setMessageText(e.target.value)}
-        />
-        <button>Add</button>
-        {/* <Editor
+      <h2 className="heading">Insert message text</h2>
+
+      <section className="login-signin-buttons">
+        <form onSubmit={onSubmitForm} className="header-flex">
+          <button>Add</button>
+          <textarea
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/NewMessage.js:40]: https://dequeuniversity.com/rules/axe/4.4/label

inputs must have associated labels

File: client/src/components/dashboard/NewMessage.js [hunk 2]
```diff
@@ -23,28 +23,35 @@ function NewMessage() {
...
-          value={messageText}
-          onChange={(e) => setMessageText(e.target.value)}
-        />
-        <button>Add</button>
-        {/* <Editor
+      <h2 className="heading">Insert message text</h2>
+
+      <section className="login-signin-buttons">
+        <form onSubmit={onSubmitForm} className="header-flex">
+          <button>Add</button>
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/NewMessage.js]: Submit buttons must be the last child of the form

File: client/src/App.css [hunk 3]
```diff
@@ -128,33 +100,26 @@ nav a.active {
...
 
-
 button {
     padding: 15px 25px;
     background-color: rgb(219, 104, 104);
     border: none;
-    border-radius: 7px;
-    display: block;
+    border-radius: 7px;   
     width: 110px;
```
Comment (SallyMcGrath) [inline client/src/App.css:116]: What happens when you scale up the interface by 200%

https://www.accessguide.io/guide/resize-text-200

File: client/src/App.css [hunk 4]
```diff
@@ -128,33 +100,26 @@ nav a.active {
...
     text-decoration: none;
     color: rgb(251, 249, 249);
     font-family: "Montserrat";
     font-weight: 700;
     font-size: 0.9rem;
     text-align: center;
-
 }
 
 .login-signin-buttons button :hover {
```
Comment (SallyMcGrath) [inline client/src/App.css:125]: 1. What is selected with this hover rule? What is inside this button?

2. What happens when someone focuses this button without hovering?

https://css-tricks.com/keyboard-only-focus-styles/

**Review by moneyinthesky — APPROVED** [review]

File: client/src/components/dashboard/Files.js [hunk 5]
```diff
@@ -39,40 +43,34 @@ function Files() {
...
-                <button onClick={() => deleteFeedback(feedback.feedback_id)}>
-                  delete feedback
-                </button>
-              </div>
-            ))}
-        </section>
         <div className="icon-heading">
-          <h1>Files</h1>
+          <h2>Files</h2>
           <img alt="files icon" src={filesIcon} />
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/Files.js:52]: is "files icon" a useful description for a user interacting with this interface without seeing it? What does files icon bring to the user experience in this context?

https://supercooldesign.co.uk/blog/how-to-write-good-alt-text

File: client/src/components/dashboard/Files.js [hunk 6]
```diff
@@ -39,40 +43,34 @@ function Files() {
...
         </div>
 
         <div className="login-signin-buttons">
           <Link to="/new-feedback">
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/Files.js:56]: (An interactive element cannot contain another interactive element. Don't wrap buttons with links. )

Buttons trigger actions or processes. Links trigger navigation only. Is this a link or a button? You may wish to add a class to apply the button styles.

File: client/src/components/dashboard/Inbox.js [hunk 7]
```diff
@@ -24,51 +25,54 @@ function Inbox() {
...
+  // console.log(messages);
+  // console.log(messages.length, messages[0]);
 
   return (
     <>
       <section>
         <div className="icon-heading">
-          <h1>Inbox</h1>
+          <h2>Inbox</h2>
           <img alt="messages icon" src={messagesIcon} />
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/Inbox.js:55]: And here, is this alt text valuable?

File: client/src/components/dashboard/Inbox.js [hunk 8]
```diff
@@ -24,51 +25,54 @@ function Inbox() {
...
-                <div key={message.message_id}>{message.message_text}</div>
-                <button onClick={() => deleteMessage(message.message_id)}>
-                  Delete
-                </button>
-              </div>
-            ))}
-        </section>
-
         <div className="login-signin-buttons">
           <Link to="/new-message">
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/Inbox.js:58]: Link/button issue here as well (btw I recognise this was in the previous code, but as it's showing up in the changelist I may as well catch it).

File: client/src/App.css [hunk 9]
```diff
@@ -128,33 +100,26 @@ nav a.active {
...
     padding: 15px 25px;
     background-color: rgb(219, 104, 104);
     border: none;
-    border-radius: 7px;
-    display: block;
+    border-radius: 7px;   
     width: 110px;
-    margin: 20px auto;
     text-decoration: none;
     color: rgb(251, 249, 249);
```
Comment (SallyMcGrath) [inline client/src/App.css:118]: What is the contrast ratio between this text color and the background color? I haven't checked but reading the rgb values my guess is that the contrast is too low to pass Lighthouse.  Do you know?

File: client/src/App.css [hunk 10]
```diff
@@ -239,6 +206,61 @@ main {
...
+    font-weight: 500;
+    color: #b55151;
+    align-self: flex-end;
+    margin-right: 30px;
+    margin-top: 50px; 
+}
+
+section {
+        padding: 30px;
+        /* margin: 30px; */
```
Comment (SallyMcGrath) [inline client/src/App.css:221]: Please remove commented out code or annotate it with purpose.

File: client/src/components/dashboard/Dashboard.js [hunk 11]
```diff
@@ -17,31 +19,46 @@ function Dashboard({ setAuth }) {
       const parseRes = await res.json();
 
       console.log(parseRes);
+
+      // setName(parseRes.username);
```
Comment (SallyMcGrath) [inline client/src/components/dashboard/Dashboard.js:23]: Please clean up commented out code, particularly duplicates.

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Hi @hachi-ops 

Thanks for this. I can see you've put a lot of work into this PR.

I have just done a minor pass over the very front end and made a few comments. My main note here is not about the code but about the commit discipline.

It's really important to work on clear communication in your codebase. Working in a team means every commit is your chance to explain and present your ideas and your work to your team, so they can clearly understand:

- what you have done
- why you have done it
- how they can help

There's little context in this PR; the commit message is brief and does not link to tickets on a board. You have committed many components in one PR and they are not clearly named in the commits. These are common habits, but good to get out of as soon as possible. Here's a great post about how to scope down PRs

https://www.netlify.com/blog/2020/03/31/how-to-scope-down-prs/

It's something worth sharing with the team and having a chat about. 

Thanks again, @hachi-ops !

**Pull request merged**